src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import duke.Deadline;
+import duke.Event;
+import duke.Task;
+import duke.Todo;
+import exception.DukeException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    static List<Task> tasks = new ArrayList<>();
+    static Boolean notBye = true;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+//        String logo = " ____        _        \n"
+//                + "|  _ \\ _   _| | _____ \n"
+//                + "| | | | | | | |/ / _ \\\n"
+//                + "| |_| | |_| |   <  __/\n"
+//                + "|____/ \\__,_|_|\\_\\___|\n";
+        String greeting = "Hello I'm Duke\n"
+                + "What can I do for you?";
+        System.out.println(greeting);
+
+        Scanner sc = new Scanner(System.in);
+        String userInput;
+        while(notBye) {
+            userInput = sc.nextLine();
+            try {
+                processUserInput(userInput);
+            }catch (DukeException e){
```

Comment:
It would be good to leave space between the brackets `{`, `}`, for layout consistency. There are several more such adjustments that could be made in the `processInput` method.